src/core_modules/capture-ui/AgeField/AgeField.component.js

```diff
@@ -211,11 +210,8 @@ class D2AgeField extends Component<Props> {
             onParseDate,
             ...passOnProps
         } = this.props;
-        const dateInputContainerClass = classNames(
-            { [defaultClasses.ageDateInputContainerHorizontal]: !isVertical },
-        );
         return (
-            <div className={dateInputContainerClass}>
```

Comment:
I removed these classes because as far as I could see, both `ageDateInputContainerHorizontal` and `dateInputContainerClass` seemed either empty classes or werent present in the css file.